hack/verify-forks.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# This script verifies forks.data file contents with our vendored directory and
+# forks we own.
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+forksData="$(dirname "${BASH_SOURCE}")/forks.data"
+
+
+declare -A depths=()
```

Comment:
OSX still has a version 3 bash 😱  `brew install bash` helps though.

Reply:
We generally do not allow associative arrays for this reason.